js/admin/tinymce.inc.js

```diff
@@ -45,31 +45,38 @@ function tinySetup(config) {
     config.selector = '.' + config.editor_selector;
   }
 
-  var default_config = {
-    selector: '.rte',
-    plugins: 'align colorpicker link image filemanager table media placeholder advlist code table autoresize',
-    browser_spellcheck: true,
-    toolbar1: 'code,colorpicker,bold,italic,underline,strikethrough,blockquote,link,align,bullist,numlist,table,image,media,formatselect',
-    toolbar2: '',
-    external_filemanager_path: baseAdminDir + 'filemanager/',
-    filemanager_title: 'File manager',
-    external_plugins: {
-      filemanager: baseAdminDir + 'filemanager/plugin.min.js'
-    },
-    language: iso_user,
-    content_style: lang_is_rtl === '1' ? 'body {direction:rtl;}' : '',
-    skin: 'prestashop',
-    menubar: false,
-    statusbar: false,
-    relative_urls: false,
-    convert_urls: false,
-    entity_encoding: 'raw',
-    extended_valid_elements: 'em[class|name|id],@[role|data-*|aria-*]',
-    valid_children: '+*[*]',
-    valid_elements: '*[*]',
-    init_instance_callback: 'changeToMaterial',
-    rel_list: [{title: 'nofollow', value: 'nofollow'}]
-  };
+  var default_config = {};
+
+  if (defaultTinyMceConfig) {
```

Comment:
```suggestion
  if (typeof defaultTinyMceConfig !== 'undefined') {
```
Otherwise 
```
VM50:1 Uncaught ReferenceError: defaultTinyMceConfig is not defined
```

Also, I think you should do a merge between the `default_config` and the overrides one.